config/v1/types_proxy.go

```diff
@@ -42,17 +42,18 @@ type ProxySpec struct {
 
 	// trustedCA is a reference to a ConfigMap containing a CA certificate bundle used
 	// for client egress HTTPS connections. The certificate bundle must be from the CA
-	// that signed the proxy's certificate and be signed for everything. trustedCA should
-	// only be consumed by a proxy validator. The validator is responsible for reading
-	// ConfigMapNameReference, validating the certificate and copying "ca-bundle.crt"
-	// from data to a ConfigMap in the namespace of an operator configured for proxy.
-	// The namespace for this ConfigMap is "openshift-config". Here is an example
+	// that signed the proxy's certificate and be signed for everything. The trustedCA
+	// field should only be consumed by a proxy validator. The validator is responsible
+	// for reading the ConfigMap referenced by ConfigMapNameReference, validating the
+	// certificate bundle and copying the bundle from "ca-bundle.crt" to a ConfigMap
```

Comment:
something in this doc should make it clear that the configmap referenced by trustedCA must contain a key named `ca-bundle.crt` as that is the only key the validator+publisher are going to care about.